socs/agents/acu/agent.py

```diff
@@ -2643,6 +2657,111 @@ def _get_sunsafe_moves(self, target_az, target_el, zero_legs_ok=True):
             return legs, None
         return legs[1:], None
 
+    @ocs_agent.param('action', choices=['open', 'close'])
+    @inlineCallbacks
+    def set_shutter(self, session, params):
+        """set_shutter(action=None)
+
+        **Task** - Request a (LAT) shutter action, wait for it to
+        complete or fail.
```

Comment:
What happens if this is run accidentally on a SAT? Worth having some early check for whether the platform supports this command?